.github/workflows/ci.yml

```diff
@@ -1,11 +1,12 @@
 name: CI
+
 on:
   push:
     branches:
       - develop
       - master
   pull_request:
-    types: [opened, synchronize, reopened]
+
```

Comment:
It looks like types need to be listed here, otherwise pull requests are not run.

Reply:
Not according to the docs:

> Note: By default, a workflow only runs when a pull_request's activity type is opened, synchronize, or reopened. To trigger workflows for more activity types, use the types keyword.

I think I have also seen this in other projects

Reply:
I have added this back in.